mne/utils/check.py

```diff
@@ -771,3 +784,28 @@ def _ensure_events(events):
         raise ValueError(
             f'events must be of shape (N, 3), got {events.shape}')
     return events
+
+
+def _infer_check_export_fmt(fmt, fname, supported_formats):
```

Comment:
Can you (very briefly) document the parameters here so one knows which data types the function expects?